tests/unit/test_arithmetic_expression.cpp

```diff
@@ -1719,3 +1719,21 @@ TEST_F(ArithmeticTest, CoalesceTest) {
 	ASSERT_EQ(0, SIValue_Compare(SI_LongVal(1), arExp->operand.constant, NULL));
 }
 
+TEST_F(ArithmeticTest, ReplaceTest) {
+	const char *query;
+	AR_ExpNode *arExp;
+
+	// Test replace in start of the string.
```

Comment:
missing a case where the old string doesn't exists in the original string
please add a test case where new string is empty ''